### DISCUSSION TITLE
Question about flashing bootloader for the nrf52840

### DISCUSSION THREAD

**Discussion by wgililland** [adafruit/Adafruit_nRF52_Bootloader#282] [Q&A]
Hello, I was looking for some help with setting up my nrf52840 DK board, I was looking at the adafruit documentation and was able to run the commands

make BOARD=pca10056 clean
make BOARD=pca10056 sd

When I get to running the flash, I get the following error:

$ MINGW32-make BOARD=pca10056 flash
mkdir: cannot create directory ‘_build/build-pca10056’: No such file or directory
Makefile:399: recipe for target '_build/build-pca10056' failed
MINGW32-make: *** [_build/build-pca10056] Error 1

I am not sure where this directory is trying to get created and was wondering if anyone had encountered a problem like this and if so how were you able to fix the problem?

**Comment by charlescuston** [comment]
For anyone else who encounters this, try:

git submodule update --init
